go.mod

```diff
@@ -1,5 +1,7 @@
 go 1.23.4
 
+toolchain go1.24.1
+
```

Comment:
Let's get https://github.com/cri-o/cri-o/pull/9051 in before this PR

Reply:
Nice, tks.